game-core/src/main/java/games/strategy/triplea/ui/MovePanel.java

```diff
@@ -1191,6 +1182,17 @@ private void updateUnitsThatCanMoveOnRoute(final Collection<Unit> units, final R
     unitsThatCanMoveOnRoute = new ArrayList<>(lastResult.getUnitsWithDependents());
   }
 
+  private Collection<Unit> getPossibleTransportsToLoad(
+      final Collection<Unit> units, final Route route) {
+    // TODO kev check for already loaded airTransports
```

Comment:
I'm with code-climate for fixing or removing this TODO, does the todo comment make any sense to anyone? I'm not sure what we should be checking and if it's not already taken care of somehow. Or is this method just incomplete and/or wrong from the get-go by not considering all transport types. Did you gain any insights @asvitkine  into this when working on this code?

Reply:
I've not had a chance to test this yet, but note that a corresponding TODO exists in MoveValidator.isLoad().

I think it's better to keep the TODO until we have a chance to validate this. This can be done by expanding test coverage for MovableUnitsFilter which I'm adding in the next PR - though my priority there is to add test cases for the other things first - like transports loading and mech infantry.